rules/sinks/third_parties/sdk/amazonaws/java.yaml

```diff
@@ -249,7 +257,7 @@ sinks:
     domains:
       - "amazonaws.com"
     patterns:
-      - "(?i)(com[.]amazonaws[.]services[.]ecr).*"
+      - "(?i)(com[.]amazonaws[.]services[.]ecr|com[.]amazonaws[.]services[.]ecrpublic).*"
```

Comment:
`ecrpublic` is anyways matched with `ecr.*` so the 2nd pattern is redundant

Reply:
removed